### DISCUSSION TITLE
What is the role of transloco.config.js?

### DISCUSSION THREAD

**Discussion by studioromeo** [ngneat/transloco#753] [Q&A] [answered]
What is the purpose of transloco.config.js? I've looked through the installation / getting started section of the docs and cloned down this library but can't see where it's used.

I ask because it contains available langs as well as the path to translations but this is also provided in the angular module / loader. So I wondered what this file was for and what the impact would be if it was misconfigured as I sense at some point it would be if someone also didn't realise there was identical config there that needed updating.

Thanks!

**Comment by shaharkazaz** [comment, marked as answer]
It's used by the [scoped lib extractor](https://jsverse.github.io/transloco/docs/tools/scope-lib-extractor) plugin and the [keys-manager](https://github.com/jsverse/transloco-keys-manager).
You can look for `getGlobalConfig` usages from the `transloco-utils` package.

**Reply by M0ns1gn0r** [reply]
@studioromeo pls mark this as an answer.

**Reply by studioromeo** [reply]
Oopsie, sorry! Marked, thanks for the nudge 🙂
